cmd/brew-test-bot.rb

```diff
@@ -975,7 +980,7 @@ def test_ci_upload(tap)
     safe_system "git", "reset", "--hard", "origin/master"
     safe_system "brew", "update"
 
-    if (pr = ENV["UPSTREAM_PULL_REQUEST"])
+    if (pr = ENV["UPSTREAM_PULL_REQUEST"] || ENV["CIRCLE_PR_NUMBER"])
```

Comment:
Pull this into a separate variable and document that the variables are for Jenkins and Circle CI respectively.